### PR TITLE
generators: fix wordy generator

### LINKED ISSUE #748: Fix wordy example to pass with the latest test data
e.g. the "just a number" test fails

To repro:
- go to the generators folder
- run `dotnet run -e wordy`
- go to the root folder
- run `./build.sh --exercise=wordy` or `.\build.ps1 -Exercise wordy`

### PR DESCRIPTION
**Pull request by ErikSchierboom**
Closes #748

### CODE REVIEW

File: exercises/wordy/Example.cs [hunk 1]
```diff
@@ -1,51 +1,49 @@
...
 
     public static int Answer(string question)
     {
-        var parsedProblem = ParseProblem(question);
-        var firstPass = Operations[parsedProblem.Operation1](parsedProblem.X, parsedProblem.Y);
-        if (parsedProblem.Operation2.Length == 0)
-            return firstPass;
-        return Operations[parsedProblem.Operation2](firstPass, parsedProblem.Z);
+        var match = Parse(question);
+        if (match is null)
```
Comment (robkeim) [inline exercises/wordy/Example.cs]: I believe `Match` always returns a value but you should check `if (!match.Success)` to determine if there was nothing to match:

> You can determine whether the regular expression pattern has been found in the input string by checking the value of the returned Match object's Success property.

https://docs.microsoft.com/en-us/dotnet/api/system.text.regularexpressions.regex.match?view=netframework-4.7.2

File: exercises/wordy/Example.cs [hunk 2]
```diff
@@ -1,51 +1,49 @@
...
-        public string Operation2;
-        public int Z;
+        switch (operand)
+        {
+            case "plus": return left + right;
+            case "minus": return left - right;
+            case "multiplied by": return left * right;
+            case "divided by": return left / right;
+            default: throw new ArgumentException();
+
```
Comment (robkeim) [inline exercises/wordy/Example.cs]: Super minor: unneeded newline here

File: exercises/wordy/Example.cs (hunk 1 above)
Comment (ErikSchierboom) [reply exercises/wordy/Example.cs]: You are right. I've removed the superfluous check.

**Comment by robkeim** [comment]
Changes look good to me, this can be merged as soon as the CI completes.

**Comment by ErikSchierboom** [comment]
Merged. Thanks for the review!

**Pull request merged**